Add roboticist backpack, satchel and duffel bag

Atakku opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Added roboticist backpack, satchel and duffel bag, to fit the roboticist jumpsuit/skirt better
Prototypes from https://github.com/ss14Starlight/space-station-14, and assets originate from tgstation

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Because its fuckin cool

## Technical details
<!-- Summary of code changes for easier review. -->
Added 3 new items, 3 new loadouts, added said loadouts to scientist, and assets for the 3 items

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="404" height="453" alt="Screenshot 2025-07-30 004014" src="https://github.com/user-attachments/assets/e79cfb89-fdf6-4200-a224-c56fcaa6730f" />
<img width="1025" height="524" alt="image" src="https://github.com/user-attachments/assets/435ac111-49de-4e70-8858-be0ec927a912" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: Added roboticist backpack, satchel and duffel bag.

imatsoup commented:
Also, these should probably be added to the robodrobe inventory. Look in Resources\Prototypes\Catalog\VendingMachines\Inventories\robodrobe.yml. Otherwise, looks good to me.

Atakku commented:
<img width="769" height="588" alt="image" src="https://github.com/user-attachments/assets/ab694f26-3f07-4435-a59b-4195e38b4474" />
Added, also moved outer coat below jumpsuit/skirt to match scidrobe's order

Atakku commented:
Im looking at where science backpack appears, i think i need to add the roboticist backpack to roboticist wardrobe
